documentation/assemblies/assembly-kafka-connect.adoc

```diff
@@ -32,13 +41,22 @@ m¦FileStreamSinkConnector
 
 |===
 
-In {ProductName}, you can use the Cluster Operator to deploy a Kafka Connect to your Kubernetes cluster.
-OpenShift also supports Kafka Connect Source-2-Image (S2I) clusters.
+To use other connector classes, you need to prepare connector images by following one of these procedures:
 
-A Kafka Connect cluster is implemented as a `Deployment` with a configurable number of workers. The Kafka Connect REST API is available on port 8083, as the `<connect-cluster-name>-connect-api` service.
+* xref:creating-new-image-from-base-{context}[] 
 
-For more information on deploying a Kafka Connect S2I cluster on OpenShift, see xref:using-openshift-s2i-create-image-str[Creating a container image using OpenShift builds and Source-to-Image].
+* xref:using-openshift-s2i-create-image-str[] (OpenShift only)
+
+The Cluster Operator can use images that you create to deploy a Kafka Connect cluster to your Kubernetes cluster.
+
+A Kafka Connect cluster is implemented as a `Deployment` with a configurable number of workers. Each individual connector's tasks are distributed among the workers in the cluster.
```

Comment:
This sentence seems a bit lost. We've already explained about tasks and workers, so all this is really adding is that it's a Deployment–what do we expect the reader to make of this fact? I think we either need to drop this paragraph, or use it to explain that scaling the `KafkaConnect.spec.replicas` increases the number of workers, which in tandem with the connector's max tasks config can be used to scale consumption or production. But maybe that's a big enough subject for a separate concept?

Reply:
I've removed the second sentence (Each individual connector's tasks are distributed among the workers in the cluster). The first sentence, about the `Deployment`, seems like useful information to me; I'd prefer to leave it in.

I've captured your notes on scaling. The priority at the moment is to get this merged in!